cmd/api/main.go

```diff
@@ -97,17 +110,17 @@ func books(w http.ResponseWriter, r *http.Request) {
 }
 
 /* Verifies if all entry fields are filled and returns a warning message if so. */
-func filledFields(newBook Book) bool {
+func filledFields(bookEntry Book) bool {
```

Comment:
Lets start some refactor here.
There is not real reason to keep this function returning a boolean.
In any of the empty fields conditions occur, just return the `errResponseBookEntryBlankFileds` error.
It will read much nicer and, eventually if we want to, we can make the function return a wide variety of errors.